docs/product/components/popovers.html

```diff
@@ -78,6 +78,11 @@
                     <td>Controller element</td>
                     <td class="p8 lh-lg">Dictates where to place the popover in relation to the reference element. By default, the placement value is <code class="stacks-code">bottom</code>. Accepted placements are <code class="stacks-code">auto</code>, <code class="stacks-code">top</code>, <code class="stacks-code">right</code>, <code class="stacks-code">bottom</code>, <code class="stacks-code">left</code>. Each placement can take an additional <code class="stacks-code">-start</code> and <code class="stacks-code">-end</code> variation.</td>
                 </tr>
+                <tr>
+                    <th scope="row"><code class="stacks-code">data-s-popover-show-on-connect="true"</code></th>
```

Comment:
Let's change to something like `data-s-popover-auto-show`. `connect` is more technically correct, but doesn't mean much to anybody who hasn't written a Stimulus controller and doesn't know that the lifecycle step is called `connect`.

Reply:
I prefer the technical precision but am fine with that.